TROUBLESHOOTING.md

```diff
@@ -53,6 +71,51 @@
 
     Spark Structured Streaming and Spark DStreams are [not supported in this release](./LIMITATIONS.md#spark-streaming) of the solution accelerator.
 
+## <a id="driver-log-eventemitter" />Databricks Driver Logs: EventEmitter Could Not Emit Lineage
+
+When reviewing the Driver logs, you see an error in the Log4j output that indicates the EventEmitter class had an exception and could not emit lineage.  You do not see any events in `OpenLineageIn` or `PurviewOut` functions either.
+
+**Solution**: This indicates a problem connecting to the Azure Function from Databricks.
+
+* Confirm spark.openlineage.url.param.code and spark.openlineage.host values are set and correct.
```

Comment:
Add single-tick markdown around the variables like `spark.openlineage.url.param.code` on lines 80 and 82

Reply:
Updated